utils/getUserData.js

```diff
@@ -0,0 +1,22 @@
+async function getUserData(endPoint, username){
+  //   endPoint ---- string ----- always add a forward slash / in front of the endpoint url
+  //   username ------ string
+  // dynamic url
+  let url = endPoint + username;
+  let option = {
+    method: "GET",
+    headers:{
+      Accept: "*",
+    },
+    Origin: "auth"

```

Comment:
This should be "originator", and should be in the `headers` object. we're basically applying our own request header, so don't let it confuse you.
This 👇 what I mean.
```
headers:{
      Accept: "*",
      originator: "auth"
    }
```